LEAF_Nexus/api/controllers/GroupController.php

```diff
@@ -67,7 +67,17 @@ public function delete($act)
         $this->index['DELETE']->register('group/[digit]/tag', function ($args) use ($group) {
             try
             {
-                $group->deleteTag((int)$args[0], $group->sanitizeInput($_GET['tag']));
+                return $group->deleteTag((int)$args[0], $_GET['tag'], $_GET['requiresAdmin']);
+            }
+            catch (Exception $e)
+            {
+                return $e->getMessage();
+            }
+        });
+        $this->index['DELETE']->register('group/[digit]/local/tag', function ($args) use ($group) {
+            try
+            {
+                return $group->deleteLocalTag((int)$args[0], $_GET['tag']);
```

Comment:
Tag GET should be run through the sanitize function

Reply:
Good spot thank you